data/hai/hai reveal 5 philosophers.txt

```diff
@@ -663,21 +663,32 @@ mission "Hai Reveal [B09] Scanning Devil-Hide"
 			`	After you both sit, Stephen Moray turns on a device and places it on the table. "Communications jammer," he explains, "so we're not heard.`
-			`	"I know that you have been working closely with the Hai lately, helping to achieve an outcome with lasting stability. I communicated with Remington and he's informed me that while it might appear to an observer that you've caused a mess, that the Hai representatives have now explained that previous stalling on their part was because some things needed to be known first. It would seem that, as per usual, there is as much going on behind the scenes as in front and you have become embroiled in it.`
-			`	"I also understand what the Hai are grappling with right now," he says. "When I was a teenager, pirates kidnapped my brother and father, and forced them to serve on pirate vessels, calling them 'indentured servants.' My father watched helplessly as my brother's disabled ship was destroyed by a Navy Frigate just as it exited hyperspace. He eventually escaped - but he's now serving his seventeenth year of a twenty-five-year sentence for piracy, even though his deeds were against his will," Moray explains as he lays out a selection of reports to emphasize his point.`
-			`	"I tell you this because it is important that you understand I am on your side in this. We have not met before, but we must trust one another. You see, the hails of the Navy: loyalty, honor, bravery - half-truths at best. We see the Navy as heroes because they are the only large organization capable of self-sacrifice. Yes, they do many good deeds, but those deeds blind us to their brutality and immorality. Not all humans are like that - many members of the Navy disagree with the barbaric rules about slavery. Some of us in the Syndicate, Free Worlds, and elsewhere, are trying to solve this problem.`
-			`	"In my time as a Syndicate executive, I have arranged for over eighty thousand slaves to be smuggled away from pirate planets to gain their freedom in Syndicate space. I have extensive contacts within the slaver community. Now I understand you have found some things, that are also of historic interest to us. For this reason I am doubly invested; so now I need to know what you know."`
+			choice
+				`	"What is this about?"`
+					goto about
+				`	"I don't appreciate how your goons treated me."`
+			`	He shrugs. "Those 'goons' were from Republic Intelligence. I have no say as to how they do their jobs. Now that you're here, let's get down to business.`
+			label about
+			`	"I know you've been working closely with the Hai lately, helping to achieve lasting stability. I communicated with Remington and he's informed me that while it might that you've caused a mess, the Hai representatives explained they were stalling while you find some information. It would seem that, as usual, there much going on behind the scenes and you have become embroiled in it.`
```

Comment:
```suggestion
			`	"I know you've been working closely with the Hai lately, helping to achieve lasting stability. I communicated with Remington and he's informed me that while it might look like you've caused a mess, the Hai representatives explained they were stalling while you find some information. It would seem that, as usual, there much going on behind the scenes and you have become embroiled in it.`
```